package.json

```diff
@@ -54,14 +54,17 @@
     "eslint-config-prettier": "^6.0.0",
```

Comment:
Please delete all the `eslint-` packages, and the `prettier` package

Reply:
Hey @JustinBeckwith! Just want to make sure I'm understanding correctly. Did you want me to delete .eslintignore, .eslintrc.json, .prettierignore and .prettierrc.js? I was using [this PR](googleapis/google-cloud-node/issues/2972
) as reference, I know Ben left those there.

Reply:
Oh noes!  Leave those files.  I'm saying the package references in the `devDependencies` section of `package.json`.  Those are transitive dependencies of `gts`, so we can delete em from here. 

Keep all the files!